modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -153,6 +153,16 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`tuningOptions`
+|`tuningOptions` specifies options for tuning the performance of Ingress Controller pods.
+
+* `headerBufferBytes` specifies how much memory is reserved in bytes for Ingress Controller connection sessions. This value must be at least `16384` if HTTP/2 is enabled for the Ingress Controller. If this field is empty, the Ingress Controller uses a default value of `32768` bytes. Setting this field is optional and not recommended because `headerBufferBytes` values that are too small can break the Ingress Controller, and `headerBufferBytes` values that are too large could cause the Ingress Controller to use significantly more memory than necessary.
+
+* `headerBufferMaxRewriteBytes` specifies how much memory should be reserved in bytes from `headerBufferBytes` for HTTP header rewriting and appending for Ingress Controller connection sessions. The minimum value for `headerBufferMaxRewriteBytes` is `4096`. `headerBufferBytes` must be greater than `headerBufferMaxRewriteBytes` for incoming HTTP requests. If this field is empty, the Ingress Controller uses a default value of `8192` bytes. Setting this field is optional and not recommended because `headerBufferMaxRewriteBytes` values that are too small can break the Ingress Controller and `headerBufferMaxRewriteBytes` values that are too large could cause the Ingres Controller to use significantly more memory than necessary.
```

Comment:
Last Ingess is missing a second S.

<shakes-fist>oh, long lines...</shakes-fist>